nnstreamer_example/custom_example_average/nnstreamer_customfilter_example_average.c

```diff
@@ -58,9 +58,7 @@ set_inputDim (void *private_data, const GstTensorFilterProperties * prop,
     const GstTensorsInfo * in_info, GstTensorsInfo * out_info)
 {
   int i;
-  pt_data *data = private_data;
 
-  assert (data);
```

Comment:
You can assert private_data directly (removing data declaration) without worrying about the release build warnings.